app/ui.js

```diff
@@ -1146,6 +1147,7 @@ const UI = {
         UI.updateSetting('max_video_resolution_x', 960);
         UI.updateSetting('max_video_resolution_y', 540);
         UI.updateSetting('framerate', 30);
+        UI.saveSetting('framerate');
```

Comment:
Most settings we don't write to local storage. Its helpful to be able to query the settings from the parent frame to keep the displays in sync. In the future perhaps:
- All settings could be saved off as part of the updateSetting ?
- A generic postMessage function could be called to share the current state of all settings to the parent frame?
- The parent frame would simply re-use the noVNC elements to adjust settings - thus not needing to monitor state